weblate/trans/views/basic.py

```diff
@@ -452,11 +452,15 @@ def show_project(request, obj):
 def show_category(request, obj):
     user = request.user
 
-    all_changes = (
-        Change.objects.for_category(obj).filter_components(request.user).prefetch()
+    last_changes = Change.objects.last_changes(
+        user, project=obj.project, category=obj
+    ).recent()
```

Comment:
`for_category` filters for changes in all components within category. These are now gone from the results, as you only filter changes having explicitly set `category`. Instead, I think that `for_category` should be adjusted to additionally include entries with matching `category`.

Reply:
Hi @nijel, I have reverted this change and fixed the `for_category`.